wemake_python_styleguide/visitors/ast/loops.py

```diff
@@ -177,6 +180,35 @@ def _check_infinite_while_loop(self, node: AnyLoop) -> None:
             if not isinstance(evaled, ast.Name) and bool(evaled):
                 self.add_violation(InfiniteWhileLoopViolation(node))
 
+    def _check_await_inside_loop(
+        self, node: AnyLoop | AnyComprehension
+    ) -> None:
+        if isinstance(node, AnyComprehension) and any(
+            generator.is_async for generator in node.generators
+        ):
+            return
+
+        bad_loops = (
```

Comment:
we use class-level constants for tuples like this. proper name would be `_forbidden_await_loops`